### PR TITLE
JBPM-7689: Focus canvas after node selection in DMN navigator dock

### PR DESCRIPTION
**Pull request by jomarko**
@manstis @hasys please have a look, one more case to check.

### CODE REVIEW

**Comment by hasys** [comment]
Nice catch! Looks good to me, thank you @jomarko

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/decision/factories/DecisionNavigatorBaseItemFactoryTest.java [hunk 1]
```diff
@@ -55,7 +56,8 @@
 import static org.mockito.Mockito.when;
 
 @RunWith(GwtMockitoTestRunner.class)
-public class DecisionNavigatorBaseItemFactoryTest {
+public class
+DecisionNavigatorBaseItemFactoryTest {
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/decision/factories/DecisionNavigatorBaseItemFactoryTest.java]: We don't usually split this on to two lines.. but it's cosmetic (I'll be surprised if the checkstyle plugin passes!)

Comment (hasys) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/decision/factories/DecisionNavigatorBaseItemFactoryTest.java]: @manstis It should pass, checkstyle doesn't check stuff like this. It checks not used imports, tab symbols, if/for/while without curly braces and things like this. All stylish moments delegated to IDE formatter.

Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/decision/factories/DecisionNavigatorBaseItemFactoryTest.java]: I am surprised how does this get in? :D will update definitely. Thanks for spotting.

**Comment by jomarko** [comment]
@manstis please merge when you have a moment.

**Pull request merged**